### PR TITLE
Merge develop (added tests) to main_v3.0

### PR DESCRIPTION
**Pull request by bikegeek**
## Pull Request Testing ##

- [ ] Describe testing already performed for these changes:</br>

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ ] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes or No]**

- [ ] Do these changes include sufficient testing updates? **[Yes or No]**

- [ ] Will this PR result in changes to the test suite? **[Yes or No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [ ] Do these changes introduce new SonarQube findings? **[Yes or No]**</br>
If **yes**, please describe:

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/appendixA.html#metplus-components-python-packages) table.
- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR definition above.
- [ ] Ensure the PR title matches the feature or bugfix branch name.
- [ ] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issue
Select: **Milestone** as the version that will include these changes
Select: **Coordinated METplus-X.Y Support** project for bugfix releases or **METcalcpy-X.Y.Z Development** project for official releases
- [ ] After submitting the PR, select the :gear: icon in the **Development** section of the right hand sidebar. Search for the issue that this PR will close and select it, if it is not already selected.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: docs/Users_Guide/installation.rst [hunk 1]
```diff
@@ -39,7 +39,7 @@ METcalcpy source code, e.g. `/User/someuser/METcalcpy`.  From this directory, ru
 
 `pip install -e .`
 
-This instructs pip to install the package based on instructios in the setup.py file located in the current directory
+This instructs pip to install the package based on instructios in the pyproject.toml file located in the current directory
```
Comment (jprestop) [inline docs/Users_Guide/installation.rst]:
```suggestion
This instructs pip to install the package based on instructions in the pyproject.toml file located in the current directory
```

File: docs/Users_Guide/installation.rst [hunk 2]
```diff
@@ -39,7 +39,7 @@ METcalcpy source code, e.g. `/User/someuser/METcalcpy`.  From this directory, ru
...
 (as indicated by the '.').  The `-e` directs pip to install the package in edit mode, so if one wishes to make changes
```
Comment (jprestop) [inline docs/Users_Guide/installation.rst:43]:
```suggestion
(as indicated by the `.`).  The `-e` directs pip to install the package in edit mode, so if one wishes to make changes
```

**Comment by bikegeek** [comment]
Merging tests for volrat created by John Sharples (originally into develop branch) into the main_v3.0

**Comment by jprestop** [comment]
> Merging tests for volrat created by John Sharples (originally into develop branch) into the main_v3.0

I checked out the main_v3.0 branch from METcalcpy and METplotpy and ran the tests to ensure all of the tests still pass.  Thankfully, they do.  But only with the addition of the command "cp test/conftest.py test/hovmoeller/".  @bikegeek Have you been able to look into why this is happening? Do you anticipate we will need to run this command in the official release instructions for WCOSS2?

**Comment by bikegeek** [comment]
@bikegeek Do you anticipate we will need to run this "cp test/conftest.py test/hovmoeller/" command in the official release instructions for WCOSS2?

**Comment by bikegeek** [comment]
@jprestop, I'm not sure.  Strangely, I can run the pytest command from the METplotpy/test directory, and the hovmoeller plot passes.  I cannot reproduce what is observed on WCOSS2.  Perhaps it would be best to provide that copy command for installation on WCOSS2.  The tests also pass within a Github action, which indicates that there is something different with the WCOSS2 environment?

**Comment by jprestop** [comment]
@bikegeek Based on [this message](https://dtcenter-projects.slack.com/archives/C02P4NVB6HK/p1731710042024729?thread_ts=1731701882.807879&cid=C02P4NVB6HK), I thought that you were able to reproduce the error on WCOSS, just not through GHA:

> it looks like pytest isn't picking up on the fixtures in the METplotpy/test/conftest.py file.  John Sharples is creating the netcdf input test file "on the fly" via a fixture (code is in conftest.py ). What's puzzling is that the other tests that use fixtures in conftest.py work OK.  The error message you see corresponds to the inability to find the fixtures used in hovmoeller.  When I copy the conftest.py file into the hovmoeller directory, everything works.
> When I query what fixtures are in a particular test path (i.e. bar), it lists the fixtures in the conftest.py.  However, when changing the test path to hovmoeller, only the built-in fixtures are found (which is what we see in the error message).  I don't understand why these tests run without error inside GHA.
> For now, can you verify that by copying the conftest.py from METplotpy/test to METplotpy/test/hovmoeller and re-run to verify that the tests run without error.

In either case, we **_can_** include the copy command in the instructions, but it seems like there should be a better longer term solution as a fix or we should put a permanent copy of conftest.py in the METplotpy/test/hovmoeller in the repository.  Thoughts?

**Comment by bikegeek** [comment]
These tests are already in the develop branch for the next coordinated release.  Since these tests were added after RC1, they shouldn't be in this coordinated release.

**Pull request closed without merging**